services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -53,6 +55,32 @@ const createCheckboxList = ({
     )
 }
 
+// This function takes a ContractAmendmentInfo and returns two lists of keys sorted by whether they are set true/false
+export function sortAmendedItems(
```

Comment:
An abstracted version would be a good resuseable helper to include in the yesno component folder.

Reply:
I left this as is for now. I think it's mostly going to be useful for further yes no's on the summary page so I bet we can adapt it when we have more.